scripts/apps/authoring-react/authoring-section/get-field-container.tsx

```diff
@@ -78,55 +78,57 @@ export function getFieldContainer(options: IGetFieldContainerOptions) {
 
     class HeaderLayout extends React.PureComponent<IEditorComponentContainerProps> {
         render() {
-            const {miniToolbar} = this.props;
+            const {miniToolbar, sectionClassNames} = this.props;
```

Comment:
revert changes that are adding `sectionClassNames` back (I removed it on purpose in develop)